pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -271,15 +271,15 @@ type clusterPatcher struct {
 	pctx networking.EnvoyFilter_PatchContext
 }
 
-func (p clusterPatcher) applyResource(hosts []host.Name, c *cluster.Cluster) *discovery.Resource {
-	cluster := p.apply(hosts, c)
+func (p clusterPatcher) patch(hosts []host.Name, c *cluster.Cluster) *discovery.Resource {
+	cluster := p.tryPatching(hosts, c)
 	if cluster == nil {
 		return nil
 	}
 	return &discovery.Resource{Name: cluster.Name, Resource: util.MessageToAny(cluster)}
 }
 
-func (p clusterPatcher) apply(hosts []host.Name, c *cluster.Cluster) *cluster.Cluster {
+func (p clusterPatcher) tryPatching(hosts []host.Name, c *cluster.Cluster) *cluster.Cluster {
```

Comment:
IMO, a `try` function should a `bool` return value?

Reply:
I am not aware about that - here the usage it tries to patch  if it patches it returns the patched cluster otherwise nil. Open to other naming if you have suggestions